ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +405,184 @@ def test_trustdomain_disable_disables_subdomain(self):
+
+        # add member
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'groupb', '--users', self.username])
+        assert 'Group name: groupb' in cmd_output.stdout_text
+        assert_str = 'Member users: {}'.format(self.username)
+        assert assert_str in cmd_output.stdout_text
+        assert 'Member groups: groupa' in cmd_output.stdout_text
+        assert 'Member of groups: groupc' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        # now check ssh on the client
+        clear_sssd_cache(client)
+        client.put_file_contents('/tmp/user_ssh_priv_key',
+                                 privkey)
+        client.run_command(['chmod', '600', '/tmp/user_ssh_priv_key'])
+
+        # after adding testuser001 to b group
+        # testuser001 will have the following memberships
+        # Member of groups: groupa, ipausers, groupb
+        # Indirect Member of group: groupc
+        try:
+            cmd = ['ssh', '-i', '/tmp/user_ssh_priv_key',
+                   '-q', '{}@{}'.format(self.username, client.hostname),
+                   'groups']
+            cmd_output = client.run_command(cmd)
+            assert self.username in cmd_output.stdout_text
+            assert 'groupa' in cmd_output.stdout_text
+            assert 'groupb' in cmd_output.stdout_text
+            assert 'groupc' in cmd_output.stdout_text
+        except Exception as e:
+            pytest.fail('Authentication using SSH key not successful', e)
+
+        # now back to server to remove member
+        cmd_output = master.run_command(['ipa', 'group-remove-member',
+                                         'groupb', '--users', self.username])
+        assert_str = 'Indirect Member users: {}'.format(self.username)
+        assert 'Group name: groupb' in cmd_output.stdout_text
+        assert 'Member groups: groupa' in cmd_output.stdout_text
+        assert 'Member of groups: groupc' in cmd_output.stdout_text
+        assert assert_str in cmd_output.stdout_text
+        assert 'Number of members removed 1' in cmd_output.stdout_text
+
+        clear_sssd_cache(master)
+
+        # now check ssh on the client again
+        # after removing testuser001 from b group
+        # testuser001 will have the following memberships
+        # Member of groups: groupa, ipausers
+        # Indirect Member of group: groupb, groupc
+        clear_sssd_cache(client)
+        try:
+            cmd = ['ssh', '-i', '/tmp/user_ssh_priv_key',
+                   '-q', '{}@{}'.format(self.username, client.hostname),
+                   'groups']
+            cmd_output = client.run_command(cmd)
+            assert self.username in cmd_output.stdout_text
+            assert 'groupa' in cmd_output.stdout_text
+            assert 'groupb' in cmd_output.stdout_text
+            assert 'groupc' in cmd_output.stdout_text
+        except Exception as e:
```

Comment:
Same comment as above, the try/catch clause doesn't add any value here.

Reply:
fixed it